bangazon-clumsy-penguins/BangazonSite
# Ajw add product

# Name - Logged In User Can Create a Product

User cannot see the "Sell a Product" link in the navbar on any page until they are logged

## Steps to Test

```
git fetch
git checkout ajw-addProduct
git pull origin ajw-addProduct
start Bangazon.sln
```

Verify that User cannot see the "Sell a Product" link in the navbar on any page until they are logged

Navigate to the Create product page by clicking the "Sell a Product" link in the navbar

Attempt to submit a blank form, you should see a validation error on all fields except, username, and product type

Add a new product with all fields filled out (Quantity and Price cannot be negative or 0)

On successful submission, you should be redirected to the Products view and see your newly added product 

##### Link to Feature Ticket
#37
